ajax/libs/leaflet-vector-layers/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "leaflet-vector-layers",
+  "filename": "lvector.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/JasonSanford/leaflet-vector-layers.git",
+    "basePath": "dist",
+    "files": [
+      "**/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/JasonSanford/leaflet-vector-layers.git"
+  },
+  "version": "1.6.0",
+  "description": "Allows to easily create vector layers from a number of geo web services, such as ArcGIS Server, Arc2Earth, GeoIQ, CartoDB and GIS Cloud. ",
+  "author": " JasonSanford",
+  "license": "ISC",
```

Comment:
@kennynaoh How do you know the license is ```ISC```? The author doesn't confirm with you in [issue](https://github.com/JasonSanford/leaflet-vector-layers/issues/63).